Source/WebCore/dom/DocumentTouch.cpp

```diff
@@ -37,19 +37,38 @@
 
 namespace WebCore {
 
-Ref<Touch> DocumentTouch::createTouch(Document& document, RefPtr<WindowProxy>&& window, EventTarget* target, int identifier, int pageX, int pageY, int screenX, int screenY, int radiusX, int radiusY, float rotationAngle, float force)
+Ref<Touch> DocumentTouch::createTouch(Document& document, RefPtr<WindowProxy>&& window, EventTarget* target, int identifier, double pageX, double pageY, double screenX, double screenY, double radiusX, double radiusY, float rotationAngle, float force)
 {
     RefPtr<LocalFrame> frame;
     if (window)
         frame = dynamicDowncast<LocalFrame>(window->frame());
     if (!frame)
         frame = document.frame();
 
+    // Match behavior from when these types were integers, and avoid surprises from someone explicitly
+    // passing Infinity/NaN.
+    if (!std::isfinite(pageX))
+        pageX = 0;
+    if (!std::isfinite(pageY))
+        pageY = 0;
+    if (!std::isfinite(screenX))
+        screenX = 0;
+    if (!std::isfinite(screenY))
+        screenY = 0;
+    if (!std::isfinite(radiusX))
+        radiusX = 0;
+    if (!std::isfinite(radiusY))
+        radiusY = 0;
+    if (!std::isfinite(rotationAngle))
+        rotationAngle = 0;
+    if (!std::isfinite(force))
+        force = 0;
+
```

Comment:
Nit: Using ternary operators here instead of if-statements might be appealing, i.e `pageX = std::isfinite(pageX) ? pageX : 0`

Reply:
@lilyspiniolas  Thanks for the review. Just updated the PR.